src/components/project/HorizontalDivider/HorizontalDivider.tsx

```diff
@@ -0,0 +1,13 @@
+import { css } from '@emotion/react';
+
+import { colors } from '~/styles/constants';
+
+export default function HorizontalDivider() {
+  return <div css={horizontalDividerCss} />;
```

Comment:
요거는 hr 태그를 쓰는게 어떨까요 !! (중요한 건 아닙니당)
```suggestion
  return <hr css={horizontalDividerCss} />;
```

Reply:
> 요거는 hr 태그를 쓰는게 어떨까요 !! (중요한 건 아닙니당)

https://github.com/depromeet/www.depromeet.com/pull/105/commits/4662b451afb2c20ff3f8c0ab807580e7648595ad -  hr 태그 잊고 살고 있었습니다 ㅋㅋㅋ 감사합니다